### PR TITLE
Update snapshot URLs

### PR DESCRIPTION
**Pull request by gummybera**
## Description

- Updates links and paths to snapshots to point to our new directory listing.

### CODE REVIEW

File: apps/core/content/nodes/guides/snapshots.md [hunk 1]
```diff
@@ -65,22 +65,23 @@ Different snapshot providers may have different instructions for using their sna
...
 
 - `beacon/pruned/`
-- `beacon/archive/`
+- `beacon/full/`
 
 Download the snapshot you would like to use with the following command:
 
 ```bash
-# $SNAPSHOT_URL example: https://storage.googleapis.com/bartio-snapshot/beacon/pruned/beacond-pruned-snapshot-202408292106.tar.lz4
+# $SNAPSHOT_URL example: https://storage.googleapis.com/bartio-snapshot/beacon%2Ffull%2Fsnapshot_beacond_full_20240913200045.tar.lz4
```
Comment (codingwithmanny) [inline apps/core/content/nodes/guides/snapshots.md]:
```suggestion
# $SNAPSHOT_URL example: https://storage.googleapis.com/bartio-snapshot/beacon/full/snapshot_beacond_full_20240913200045.tar.lz4
```

File: apps/core/content/nodes/guides/snapshots.md [hunk 2]
```diff
@@ -106,7 +107,7 @@ Execution client snapshots coming soon. They are not required to run a node.
 
 You can verify the snapshot against the checksum to ensure the snapshot downloaded is valid. The checksum is a hash of the snapshot file that can be used to verify the snapshot's integrity.
 
-Checksum files are provided by the Berachain Foundation with the file extension `.sha256` added to the end of the snapshot file name. For example, if you would like the sha256sum for the snapshot file example above, it is `https://storage.googleapis.com/bartio-snapshot/beacon/pruned/beacond-pruned-snapshot-202408292106.tar.lz4.sha256`.
+Checksum files are provided by the Berachain Foundation with the file extension `.sha256` added to the end of the snapshot file name. For example, if you would like the sha256sum for the snapshot file example above, it is `https://storage.googleapis.com/bartio-snapshot/beacon%2Ffull%2Fsnapshot_beacond_full_20240913200045.tar.lz4.sha256`.
```
Comment (codingwithmanny) [inline apps/core/content/nodes/guides/snapshots.md]:
```suggestion
Checksum files are provided by the Berachain Foundation with the file extension `.sha256` added to the end of the snapshot file name. For example, if you would like the sha256sum for the snapshot file example above, it is `https://storage.googleapis.com/bartio-snapshot/beacon/full/snapshot_beacond_full_20240913200045.tar.lz4.sha256`.
```

**Comment by gummybera** [comment]
@codingwithmanny thanks, changes resolved!

EDIT: Added one more change - file path in docs [updated](https://github.com/berachain/docs/pull/293/commits/eb0be7c0fd58efb5c4f05ced8f24381bbe4afd1f) to reflect new nomenclature.

**Comment by codingwithmanny** [comment]
And now for the last step @gummybera let me know if you need help with this.

https://github.com/berachain/docs/blob/main/CONTRIBUTING.md#versioning

**Comment by gummybera** [comment]
> And now for the last step @gummybera let me know if you need help with this.
> 
> https://github.com/berachain/docs/blob/main/CONTRIBUTING.md#versioning

Version updated, let me know if it looks good!

**Review by codingwithmanny — APPROVED** [review]
LGTM

**Pull request merged**